elastic/kibana
# Rename "scheduled actions" to "generated actions"

Fixes #129122

In this PR, I'm renaming some changes we did in 8.2 that used "scheduled actions" terminology and align it with "generated actions" terminology before we have to consider backwards compatibility.

## To verify

1. Create a rule that generates actions
2. Navigate to the rule details page
3. Verify that "Generated actions" and "Triggered actions" column get populated accordingly in the run history tab.
